src/test/java/org/broadinstitute/http/nio/HttpSeekableByteChannelUnitTest.java

```diff
@@ -209,80 +221,67 @@ public void testClose() throws Exception {
     }
 
     @Test
-    public void testPositionOnGoogleFile() throws IOException {
-        URL url = new URL(LARGE_FILE_ON_GOOGLE);
-        try(URLSeekableByteChannel channel = new URLSeekableByteChannel(url)) {
+    public void testPositionOnGoogleFile() throws IOException, URISyntaxException {
+        URI URI = new URI(LARGE_FILE_ON_GOOGLE);
+        try (SeekableByteChannel channel = getChannel(URI)) {
             channel.position(10000);
             channel.read(ByteBuffer.allocate(100));
         }
     }
 
-    @Test(timeOut=10_000L) //time out if it's taking over 10 seconds to read, that indicates we're probably not skipping correctly
-    public void testReadAtEndOfLargeFileIsFast() throws IOException {
-        URL url = new URL(LARGE_FILE_ON_GOOGLE);
-        try(URLSeekableByteChannel channel = new URLSeekableByteChannel(url)) {
+    @Test(timeOut = 10_000L)
+    //time out if it's taking over 10 seconds to read, that indicates we're probably not skipping correctly
+    public void testReadAtEndOfLargeFileIsFast() throws IOException, URISyntaxException {
+        URI URI = new URI(LARGE_FILE_ON_GOOGLE);
+        try (SeekableByteChannel channel = getChannel(URI)) {
             long size = channel.size();
             Assert.assertEquals(size, 31710132189L);
             channel.position(size - 200);
             Assert.assertTrue(channel.read(ByteBuffer.allocate(100)) > 0);
```

Comment:
Can you do a test on the large file that checks the actual bytes returned (for some small snippet)?